File: 2293-minimum-number-of-steps-to-make-two-strings-anagram-ii/minimum-number-of-steps-to-make-two-strings-anagram-ii.py
class Solution:
    def minSteps(self, s: str, t: str) -> int:
        arr = [0]*26
        for c in s:
            arr[ord(c)-ord('a')] += 1
        for c in t:
            arr[ord(c)-ord('a')] -= 1
        res = 0
        for i in range(26):
            res += abs(arr[i])
        return res

File: 2293-minimum-number-of-steps-to-make-two-strings-anagram-ii/minimum-number-of-steps-to-make-two-strings-anagram-ii.cpp
class Solution {
public:
    int minSteps(string s, string t) {
        int arr[26] = {0};
        for (char c: s) arr[c-'a']++;
        for (char c: t) arr[c-'a']--;
        int res =0;
        for (int i=0; i<26; i++) res += std::abs(arr[i]);
        return res;
    }
};